profile/load_request.go

```diff
@@ -5,25 +5,25 @@ import (
 )
 
 // LoadRequestOptions loads request configurations from the profile names passed in. It returns
-// an error if it doesn't find a request configuration with the specified name.
+// an error if it doesn't find a request configuration with the specified name. It will return the
+// last request options found, if the same name is found in multiple profiles.
 func LoadRequestOptions(requestNameToFind string, profiles []string) (*RequestOptions, error) {
+	result := make([]RequestOptions, 0)
 	for _, profileName := range profiles {
 		profileOptions, err := LoadProfile(profileName)
 		if err != nil {
 			return nil, err
 		}
 
-		var result *RequestOptions
 		for requestName, requestConfiguration := range profileOptions.RequestOptions {
 			if requestNameToFind == requestName {
-				result = &requestConfiguration
```

Comment:
I think it doesn't work because `result` now points to the memory address of `requestConfiguration` not it's value

![image](https://user-images.githubusercontent.com/8136030/35014125-8c4d7f44-fadd-11e7-854d-be8b8ddb0509.png)

https://play.golang.org/p/Ge1Nmc0uECn